CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.4.3 (Next)
 
+* [#49](https://github.com/dblock/strava-ruby-client/pull/51): Added a test for parsing big/long Integer IDs - [@simonneutert](https://github.com/dblock/strava-ruby-client/pull/51).
```

Comment:
Nitpick: this description is a bit misleading, it implies `Integer`, but really it's lowercase integer :) I would say "Ensure support for large integer IDs"